### DISCUSSION TITLE
Authentifizierung auch ohne https?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2691] [General]
Müsste eine Authentifizierung nicht auch bei nicht sicherer Verbindung möglich sein? Sprich einfach PIN?
Die kann ich aber nur auswählen, wenn ich sichere Verbindung anklicke.

**Comment by Zefau** [comment]
Ohne sichere Verbindung wird alles im Klartext übertragen.

**Comment by mcuiobroker** [comment]
Ich möchte ja nur ein Password für das Frontend. Wenn ich jetzt "sichere Verbindung" wähle und für Frontend ein Passwort vergebe bekomme ich einen Fehler.

![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/6996c051-af47-4673-8b50-42a53a16c1a8)
Dabei ist mir aufgefallen, dass er die "Sichere Verbindung" wieder gelöscht hat? Haken raus, obwohl mit gespeichert.
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/d3e0c989-a0fb-4e29-9581-8810e0f7df2c)
